kubernetes/apps/kube-system/cilium/app/cilium-l2.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# https://docs.cilium.io/en/latest/network/l2-announcements
+apiVersion: cilium.io/v2alpha1
+kind: CiliumL2AnnouncementPolicy
+metadata:
+  name: policy
+spec:
+  loadBalancerIPs: true
+  # NOTE: This might need to be set if you have more than one active NIC on your nodes
```

Comment:
Is it unsafe to leave these commented out? Can we just enforce it?

Reply:
It's a pretty big can of worms because nodes might not have the same interface names and it would also need to be a user supplied config. I'm not ready to add this in yet.